### PR TITLE
Fix bug with textbox clipping under small width and height

### LINKED ISSUE #641: TextBox right border is clipped w/ smaller Width set
**Describe the bug**

TextBox right border gets clipped if a small enough (e.g. <=60) width is set:

image

Previously reported here: https://developercommunity.visualstudio.com/content/problem/535762/uwp-textbox-width-clipping.html

**Steps to reproduce the bug**

1. Create a page and put `<TextBox AcceptsReturn="True" Width="60" />` in it
2. Visualize it

**Expected behavior**

Right border is visible.

**Version Info**

NuGet package version: 

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)                   |    |
| October 2018 Update (17763)   | X   |
| April 2018 Update (17134)        |    |
| Fall Creators Update (16299)     |    |
| Creators Update (15063)           |    |
| Anniversary Update (14393)      |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | X   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**

### PR DESCRIPTION
**Pull request by chingucoding**
## Summary
Fix a bug with the TextBox component where it would start clipping when width and height is set to small values.

## Motivation
Fixes #641.

## Screenshots
image

### CODE REVIEW

File: dev/CommonStyles/TextBox_themeresources.xaml [hunk 1]
```diff
@@ -292,8 +292,8 @@
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                             contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}"
                             Control.IsTemplateFocusTarget="True"
-                            MinWidth="{ThemeResource TextControlThemeMinWidth}"
```
[the remark this answers is not in the document]
Comment (chingucoding) [reply dev/CommonStyles/TextBox_themeresources.xaml]: Makes sense, I will add the appropriate setters.

**Comment by adrientetar** [comment]
@chingucoding Thanks! In the same vein there's #689 if you want to look at that one.

**Comment by chingucoding** [comment]
> @chingucoding Thanks! In the same vein there's #689 if you want to look at that one.

@adrientetar I will take a look at it, thank you for pointing that issue out.

**Pull request merged**